packages/docusaurus-utils-validation/src/validationSchemas.ts

```diff
@@ -167,3 +170,122 @@ export const ContentVisibilitySchema = JoiFrontMatter.object<ContentVisibility>(
       "Can't be draft and unlisted at the same time.",
   })
   .unknown();
+
+export const FrontMatterAuthorErrorMessage =
+  '{{#label}} does not look like a valid blog post author. Please use an author key or an author object (with a key and/or name).';
+
+export const FrontMatterLastUpdateErrorMessage =
+  '{{#label}} does not look like a valid last update object. Please use an author key with a string or a date with a string or Date.';
+
+export const FrontMatterLastUpdateSchema = Joi.object({
+  author: Joi.string(),
+  date: Joi.date().raw(),
+})
+  .or('author', 'date')
+  .messages({
+    'object.missing': FrontMatterLastUpdateErrorMessage,
+    'object.base': FrontMatterLastUpdateErrorMessage,
+  });
+
+const BlogPostFrontMatterAuthorSchema = Joi.object({
```

Comment:
This is specific to the blog plugin only 😅 

I only wanted you to move `FrontMatterLastUpdateSchema` and other shared schemas between docs and blog here. Not plugin-specific validation code.

All the code below this line should be moved back to its original location